features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/postgresql.sql

```diff
@@ -178,16 +178,16 @@ CREATE TABLE IF NOT EXISTS IDN_OAUTH2_DEVICE_FLOW (
             USER_DOMAIN VARCHAR(50),
             IDP_ID INTEGER,
             PRIMARY KEY (DEVICE_CODE),
+            UNIQUE (CODE_ID),
```

Comment:
Haven't we already declared CODE_ID as UNIQUE in the next line?

Reply:
That defines 'CODE_ID, USER_CODE, DEVICE_CODE' together as unique

Reply:
In this case, do we need to make the combination of CODE_ID, USER_CODE & DEVICE_CODE as unique since CODE_ID is always unique this combination will be always unique right?

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/2682